cdc/entry/mounter.go

```diff
@@ -119,6 +119,10 @@ func (m *Mounter) mountRowKVEntry(row *rowKVEntry) (*model.DML, error) {
 }
 
 func (m *Mounter) mountIndexKVEntry(idx *indexKVEntry) (*model.DML, error) {
+	// skip set index KV
+	if !idx.Delete {
+		return nil, nil
```

Comment:
Please add a test case to help reviewers understand what this change tried to fix.

Reply:
this change fixes nothing. We translate setting row key to `replace` query, so we needn't build delete dml for setting index key.

Reply:
see:https://docs.google.com/document/d/1D1IqIs_q0y0YSnu6hJgKZO4lkCkDqOexDxzMdK-7NRU/edit#